internal/migrations/observability/api/pipelines.go

```diff
@@ -96,7 +96,7 @@ func generatePipelineFilters(loggingPipelineSpec logging.PipelineSpec, createdFi
 		}
 	}
 	if len(loggingPipelineSpec.Labels) != 0 {
-		openshiftLabelFilterName := fmt.Sprintf("filter-%s-%s", loggingPipelineSpec.Name, string(obs.FilterTypeOpenshiftLabels))
+		openshiftLabelFilterName := fmt.Sprintf("filter-%s-%s", loggingPipelineSpec.Name, openshiftLabelsFilterName)
```

Comment:
why not downcase the name instead of introducing an unnecessary constant?

Reply:
i think that have readable name make sense here